src/irr/asset/CGLSLVirtualTexturingBuiltinIncludeLoader.h

```diff
@@ -66,12 +59,17 @@ layout(set=_IRR_VT_DESCRIPTOR_SET, binding=_IRR_VT_INT_VIEWS_BINDING) uniform is
 #endif
 #if _IRR_VT_UINT_VIEWS_COUNT
 layout(set=_IRR_VT_DESCRIPTOR_SET, binding=_IRR_VT_UINT_VIEWS_BINDING) uniform usampler2DArray uphysicalTileStorageFormatView[_IRR_VT_UINT_VIEWS_COUNT];
+#endif
```

Comment:
you want to check that value is >0 (not that they're defined, they always will be)

Reply:
it's `#if`, not `#ifdef`

Reply:
ah ok